lib/Cake/Cache/Engine/RedisEngine.php

```diff
@@ -228,7 +228,7 @@ public function clearGroup($group) {
  * Disconnects from the redis server
  */
 	public function __destruct() {
-		if (!$this->settings['persistent']) {
+		if (empty($this->settings['persistent'])) {
```

Comment:
Perhaps we should also add:
```suggestion
		if (empty($this->settings['persistent']) && !is_null($this->_Redis)) {
```

To prevent an attempt at closing an uninitialized redis instance, as documented in the PHP Fatal Error in issue #13001.

Reply:
Sounds like a good idea to me.

Reply:
Done!